lib/features/user/presentation/pages/user_pages.dart

```diff
@@ -329,10 +329,13 @@ class UserStatusPageState extends State<UserStatusPage> {
       mainAxisSize: MainAxisSize.max,
       mainAxisAlignment: MainAxisAlignment.spaceBetween,
       children: [
-        _buildOperationName(operation, context),
-        Spacer(),
-        _buildActivityStatus(context),
-        Spacer(),
+        Expanded(
+          child: _buildOperationName(operation, context),
+    ),
```

Comment:
Please ensure that dartfm is enabled. See https://flutter.dev/docs/development/tools/formatting